### PR TITLE
Update project-walkthrough.md

### PR DESCRIPTION
**Pull request by thomasr8**
# Description
replaced legacy images with next-gen images.

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

### CODE REVIEW

File: jekyll/_cci2/project-walkthrough.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/python:3.6.2-stretch-browsers
+      - image: cimg/python:3.6.2-stretch-browsers
```
Comment (rosieyohannan) [inline jekyll/_cci2/project-walkthrough.md:39]: This is extracted from the demo app linked to in the intro text. AFAIU the demo app will need updating to use a next gen image first. Also Browser teating using a different image variant now, as mentioned in the migration guide: https://discuss.circleci.com/t/legacy-convenience-image-deprecation/41034#browser-testing-4

I feel like some of these changes are a little more involved that straight swap to `cimg`.

Comment (rosieyohannan) [reply jekyll/_cci2/project-walkthrough.md:39]: @felicianotech ^^

Comment (rosieyohannan) [reply jekyll/_cci2/project-walkthrough.md:39]: https://github.com/CircleCI-Public/circleci-demo-python-flask/blob/da7d6585eeac60722e04b10f612acb327e9cd72c/.circleci/config.yml#L30

**Review by felicianotech — CHANGES_REQUESTED** [review]
Only namespace changes were made here. We still need to do tags.

**Review by felicianotech — APPROVED** [review]

**Pull request merged**